k8s/test/e2e/local-assembly-container/Dockerfile.assembly

```diff
@@ -0,0 +1,26 @@
+# Copyright 2020-2022 Alibaba Group Holding Limited.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#FROM vineyardcloudnative/vineyard-python:latest
+FROM docker.pkg.github.com/v6d-io/v6d/vineyard-python-dev:latest
+
+RUN pip3 install bash
```

Comment:
Why do you need to install another bash using pip? We are using a ubuntu-based container.